src/cli/plugin/plugin_lister.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+export function list(settings, logger) {
+  const files = fs.readdirSync(settings.pluginDir);
+  files
+  .filter(function (file) {
+    return file[0] !== '.';
```

Comment:
Oh, I totally misunderstood what this filter was even doing.  Remove it if you think that's OK, but you definitely should *not* try to take my advice on this.

If you do need to keep the check, I'd recommend using lodash's `startsWith` function instead, which would just make it a bit more clear that we're dealing with the first character of a string rather than the first element of an array.